.github/workflows/tidy-modern.yml

```diff
@@ -0,0 +1,35 @@
+name: tidy-modern
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  tidy:
+    runs-on: ubuntu-latest
+    container:
+      image: pointcloudlibrary/env:18.04
+    
+    steps:
+    - name: Install git
+      run: |
+          add-apt-repository ppa:git-core/ppa
+          apt-get update
+          apt-get install -y git
+          `which git` --version
+
+    - uses: actions/checkout@v2
+
+    - name: Run clang-tidy
+      run: |
+          wget https://apt.llvm.org/llvm.sh
+          chmod +x llvm.sh
+          ./llvm.sh 10
+          apt-get update
+          apt-get install -y clang-tidy-10 clang-tools-10 python-pip
```

Comment:
Might make sense to add step from [this](https://apt.llvm.org/) in the env Dockerfile, and then use env:20.04 which has llvm-10

Reply:
Is env:20.04 the new baseline for CI?

Reply:
Apparently, 20.04 is not rolled out yet:

> Ubuntu-latest workflows will use Ubuntu-20.04 soon. For more details, see https://github.com/actions/virtual-environments/issues/1816